### PR TITLE
Rewrite app gallery

### PR DESCRIPTION
**Pull request by itsjoeoui**

### CODE REVIEW

File: src/utils/GalleryLoader.ts [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    {
+      title: "Intro to LOOPS",
+      description: "Generate a random number once",
+      imagePath: "../../assets/appGallery/introToLoops.png",
+      youtubeLink: "",
+      appPath: "randomNumber",
+      relevantNodes: [
+        {
+          name: "LOOP",
+          docs: "https://docs.flojoy.ai/nodes/LOGIC_GATES/LOOPS/LOOP/",
```
Comment (JeffDotPng) [inline src/utils/GalleryLoader.ts:14]: Why are the docs links here if the relevant node badges aren't clickable?
Also you should add more relevant nodes for each app

**Review by JeffDotPng — COMMENTED** [review]
image
"App Gallery" title could probably be smaller and left aligned (or just get rid of it as a whole, the modal is self explanatory), need a bit of margin between the search bar and the 'Fundamentals' section also

File: src/utils/GalleryLoader.ts (hunk 1 above)
Comment (itsjoeoui) [reply src/utils/GalleryLoader.ts:14]: oops I forgot to add it, let me do that

File: src/utils/GalleryLoader.ts [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+      youtubeLink: "",
+      appPath: "prophet",
+      relevantNodes: [
+        {
+          name: "PROPHET_PREDICT",
+          docs: "https://docs.flojoy.ai/nodes/AI_ML/PREDICT_TIME_SERIES/PROPHET_PREDICT/",
+        },
+      ],
+    },
+    // {
```
[the remark this answers is not in the document]
Comment (itsjoeoui) [reply src/utils/GalleryLoader.ts:75]: this app does not exist haha

**Pull request merged**